common/flatpak-run.c

```diff
@@ -3856,7 +3856,6 @@ flatpak_run_setup_base_argv (GPtrArray      *argv_array,
 
   add_args (argv_array,
             "--unshare-pid",
-            "--unshare-user-try",
```

Comment:
I thought that the way `--unshare-user-try` worked (at least a while ago) was to not try user namespaces when bwrap is setuid. Wouldn't it be better to do such a fix in bubblewrap? (check if `getuid() != geteuid()` and ignore requests for user namespaces in that case)

BTW, Alex's blog suggests not using user namespaces when bwrap is setuid - https://blogs.gnome.org/alexl/2017/01/18/the-flatpak-security-model-part-1-the-basics/.